homeassistant/components/amazon_rekognition/image_processing.py

```diff
@@ -0,0 +1,128 @@
+"""
+Platform that will perform object detection.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/image_processing/amazon_rekognition
+"""
+import base64
+import json
+import logging
+import time
+
+import voluptuous as vol
+
+from homeassistant.core import split_entity_id
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.image_processing import (
+    PLATFORM_SCHEMA, ImageProcessingEntity, CONF_SOURCE, CONF_ENTITY_ID,
+    CONF_NAME)
+
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_REGION = 'region_name'
+CONF_ACCESS_KEY_ID = 'aws_access_key_id'
+CONF_SECRET_ACCESS_KEY = 'aws_secret_access_key'
+CONF_TARGET = 'target'
+DEFAULT_TARGET = 'Person'
+
+DEFAULT_REGION = 'us-east-1'
+SUPPORTED_REGIONS = ['us-east-1', 'us-east-2', 'us-west-1', 'us-west-2',
+                     'ca-central-1', 'eu-west-1', 'eu-central-1', 'eu-west-2',
+                     'eu-west-3', 'ap-southeast-1', 'ap-southeast-2',
+                     'ap-northeast-2', 'ap-northeast-1', 'ap-south-1',
+                     'sa-east-1']
+
+REQUIREMENTS = ['boto3 == 1.9.69']
```

Comment:
Please put it into `aws` component, and see if you can use `aiobotocore` instead of `boto3`

Reply:
Excellent thanks @awarecan, will do. Cheers